app/components/samples/table_component.html.erb

```diff
@@ -228,34 +219,10 @@
     </table>
   <% end %>
   <% if @has_samples %>
-    <%= render PaginationComponent.new(
-      info: helpers.pagy_info(@pagy),
-      prev_url:
-        (
-          if @pagy.prev
-            helpers.pagy_url_for(@pagy, @pagy.prev).gsub(
-              "samples.turbo_stream",
-              "samples",
-            )
-          else
-            nil
-          end
-        ),
-      next_url:
-        (
-          if @pagy.next
-            helpers.pagy_url_for(@pagy, @pagy.next).gsub(
-              "samples.turbo_stream",
-              "samples",
-            )
-          else
-            nil
-          end
-        ),
-      data: {
-        turbo_action: "replace",
-      },
-    ) %>
+    <div class="flex items-center justify-between">
+      <%= render Viral::Pagy::LimitComponent.new(@pagy, item: t(".limit.item")) %>
```

Comment:
Update `t(".limit.item")` to `t(".limit.item", count: @pagy.limit)`. We will use this to either display the singular type  (Sample, ... etc) or pluralized type (Samples, ... etc). With this change we will need to make a few changes to the samples table_component locale files and a change to the limit component html

Reply:
Updated in e3d2f6e